File: python/99_calculate_distance.py
import math
import re
import sys

with open(sys.argv[1], 'r') as fh:
    for line in fh.readlines():
        line = line.strip()
        if line == '':
            continue

        (x0, y0, x1, y1) = [int(v) for v in re.findall(r'-?\d+', line)]

        print int(math.sqrt(math.pow(x1 - x0, 2) + math.pow(y1 - y0, 2)))

File: perl/99_calculate_distance.pl
open my $fh, '<', $ARGV[0];

for $line ( <$fh> ) {
	$line =~ s/[\n\r]+//g;
	next if $line eq '';

	$line =~ /\((\-?\d+), (\-?\d+)\) \((\-?\d+), (\-?\d+)\)/;

	( my $x0, my $y0, my $x1, my $y1 ) = ( $1, $2, $3, $4 );

	print sqrt( ( $x1 - $x0 ) ** 2 + ( $y1 - $y0 ) ** 2 ), "\n";
}

close $fh;

exit 0;

